java-checks/src/test/java/org/sonar/java/checks/InstanceOfPatternMatchingCheckTest.java

```diff
@@ -30,21 +30,12 @@ class InstanceOfPatternMatchingCheckTest {
   @Test
   void test() {
     CheckVerifier.newVerifier()
+      .withJavaVersion(20, true)
       .onFile(testSourcesPath("checks/InstanceOfPatternMatching.java"))
       .withCheck(new InstanceOfPatternMatchingCheck())
-      .withJavaVersion(16)
       .verifyIssues();
   }
 
-  @Test
```

Comment:
it seems that these features require Preview enabled, so I don't think it works on previous, thus I'm removing the test